config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglogmetrics.logging.cnrm.cloud.google.com.yaml

```diff
@@ -1,63 +1,69 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    cnrm.cloud.google.com/version: 0.0.0-dev
-  creationTimestamp: null
-  labels:
-    cnrm.cloud.google.com/dcl2crd: "true"
-    cnrm.cloud.google.com/managed-by-kcc: "true"
-    cnrm.cloud.google.com/stability-level: stable
-    cnrm.cloud.google.com/system: "true"
+    controller-gen.kubebuilder.io/version: v0.15.0
   name: logginglogmetrics.logging.cnrm.cloud.google.com
 spec:
   group: logging.cnrm.cloud.google.com
   names:
     categories:
     - gcp
     kind: LoggingLogMetric
+    listKind: LoggingLogMetricList
     plural: logginglogmetrics
     shortNames:
     - gcplogginglogmetric
     - gcplogginglogmetrics
     singular: logginglogmetric
-  preserveUnknownFields: false
```

Comment:
Oooh good call.  There's something annoying here like an upgraded CRD from v1beta1 -> v1 has this as true, so we need to set it.  I can't remember the details TBH.  But we should try to make sure we write `preserveUnknownFields: false`